masterbase/steam.py

```diff
@@ -79,27 +80,27 @@ class Filters:
     )
 
     def __init__(
-        self,
-        dedicated: bool | None = None,
-        secure: bool | None = None,
-        gamedir: str | None = None,
-        mapname: str | None = None,  # inconsistency here because polluted namespace with python `map`
-        linux: bool | None = None,
-        password: bool | None = None,
-        empty: bool | None = None,
-        full: bool | None = None,
-        proxy: bool | None = None,
-        appid: int | None = None,
-        napp: int | None = None,
-        noplayers: bool | None = None,
-        white: bool | None = None,
-        gametype: list[str] | str | None = None,
-        gamedata: list[str] | str | None = None,
-        gamedataor: list[str] | str | None = None,
-        name_match: str | None = None,
-        version_match: str | None = None,
-        collapse_addr_hash: bool | None = None,
-        gameaddr: str | None = None,
+            self,
+            dedicated: bool | None = None,
+            secure: bool | None = None,
+            gamedir: str | None = None,
+            mapname: str | None = None,  # inconsistency here because polluted namespace with python `map`
```

Comment:
nice thank you I was struggling for a var name

Reply:
I didn't actually do the rename, it's just showing changes to this file because I let PyCharm reformat the file in accordance with PEP8